cmd/storj-sim/network.go

```diff
@@ -112,14 +112,17 @@ func newNetwork(flags *Flags) (*Processes, error) {
 
 	processes := NewProcesses()
 	var (
-		configDir        = flags.Directory
-		host             = flags.Host
-		gatewayPort      = 9000
-		bootstrapPort    = 9999
-		satellitePort    = 10000
-		storageNodePort  = 11000
-		consolePort      = 10100
-		bootstrapWebPort = 10010
+		configDir              = flags.Directory
+		host                   = flags.Host
+		gatewayPort            = 9000
```

Comment:
I've been wondering whether there's a nice consistent way how to assign these ports. Currently they seem quite messy and disjointed. Maybe everything something like bootstrap ports start at `9900`, satellites start at `10XXY`, where XX is the satellite index and Y is a subsystem (gateway at `10XX9`), storage nodes at `11XXY`? Or something like that? No need to do in this PR, but just something to think about.